CIScripts/Test/Tests/Functional/DockerDriverNetworking.Tests.ps1

```diff
@@ -0,0 +1,52 @@
+Describe "Docker driver networking" {
+    Context "conflicting local IPs" {
+        It "can create containers for different tenants with same IP" -Pending {
```

Comment:
Is there something like "Skipped" in pester? (I guess not, so we're using pending as skipped)

Reply:
We see that tests without `Should` are already marked as "Inconclusive". What's the reason for `-Pending` then?

Reply:
Skipped is for tests that are implemented, but are to be skipped, e.g. because they are too long or we don't want to run them in CI.

I don't like the "inconclusive as default" behaviour of pester. It should be "error by default".

-Pending is there to signify that I haven't forgotten to write the test.